.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,33 @@
+name: Maintenance
```

Comment:
just a suggestion ... perhaps a more specific name/description like "Close Stale Issues and PRs" or "Backlog Maintenance"